### ISSUE TITLE
How I can run it?

### ISSUE THREAD

**Issue by lequynhhoa** [Arct1k/mbxd#1]
C

**Comment by Arct1k** [comment]
its decomplite core only . 
you can use it in you software with your GUI or etc.

**Comment by lequynhhoa** [comment]
Yes. I want to test decomplite core. Could you show me? Thank you



Hoa Le

*Le Thi Quynh Hoa*

GreenField consulting & development Ltd.,

*Add:* Suite 705, 14 Tran Hung Dao, Cotevina Building Hoan Kiem, Hanoi,
Vietnam

*Mobile*: (+84) 986 087 781

*Email*: le.thi.quynh.hoa@gmail.com

**Comment by sancarn** [comment]
Are you trying to use it in Visual Studio?

Something that may help is demonstrated [here](https://stackoverflow.com/questions/29896246/how-to-use-namespaces-properly-in-vb-net) which has the simple fix to append the word 'global' to prefix all namespaces:

Before:

```
using MBXD.Decompiler
namespace MBXD.Decompiler
{

}
```

After:

```
using MBXD.Decompiler
namespace global.MBXD.Decompiler
{

}
```

This will actually link all the files together giving you direct access to the MBXD.Decompiler objects. Using the decompiler from here, I am yet to figure out but looking at the code it shouldn't be too difficult to figure out.

**Comment by sdvoeglazov** [comment]
can get a compiled version?
